TransferList/TransferList.js

```diff
@@ -587,10 +641,10 @@ const TransferListBase = kind({
 				<Cell className={componentCss.listButtons}>
 					{!moveOnSpotlight ?
 						<>
-							<Button onClick={moveIntoSecondAll} onSpotlightUp={handleSpotlightBounds} size="small">{'>>>'}</Button>
-							<Button disabled={!(selectedItems.find((item) => item.list === "first"))} onClick={moveIntoSecondSelected} size="small">{'>'}</Button>
-							<Button disabled={!(selectedItems.find((item) => item.list === "second"))} onClick={moveIntoFirstSelected} size="small">{'<'}</Button>
-							<Button onClick={moveIntoFirstAll} size="small">{'<<<'}</Button>
+							<Button disabled={disabled || (secondListMaximumCapacity || firstListMinimumCapacity) !== undefined} onClick={moveIntoSecondAll} onSpotlightUp={handleSpotlightBounds} size="small">{'>>>'}</Button>
+							<Button disabled={!(selectedItems.find((item) => item.list === "first")) || disabled} onClick={moveIntoSecondSelected} size="small">{'>'}</Button>
+							<Button disabled={!(selectedItems.find((item) => item.list === "second")) || disabled} onClick={moveIntoFirstSelected} size="small">{'<'}</Button>
+							<Button disabled={disabled || (firstListMaximumCapacity || secondListMinimumCapacity) !== undefined} onClick={moveIntoFirstAll} size="small">{'<<<'}</Button>
```

Comment:
same logic here. we don't need to use undefined. please remove from eslint-disable rules